sdk/storage/Azure.Storage.Blobs/src/autorest.md

```diff
@@ -4,11 +4,29 @@ Run `dotnet build /t:GenerateCode` to generate code.
 
 ``` yaml
 input-file:
-    - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/b50bcdde18465bbf04937a9ee73607753d24d65c/specification/storage/data-plane/Microsoft.BlobStorage/preview/2020-10-02/blob.json
+    - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/c7e65d6c8db47714c21e41adf2ec60e64afae776/specification/storage/data-plane/Microsoft.BlobStorage/preview/2020-10-02/blob.json
 # https://github.com/Azure/autorest/issues/4075
 skip-semantics-validation: true
 ```
 
+``` yaml
+directive:
+- from: swagger-document
+  where: $["x-ms-paths"]
+  transform: >
+    for (const property in $)
+    {
+        if (property.includes('/{containerName}/{blob}'))
+        {
+            $[property]["parameters"] = $[property]["parameters"].filter(function(param) { return (typeof param['$ref'] === "undefined") || (false == param['$ref'].endsWith("#/parameters/ContainerName") && false == param['$ref'].endsWith("#/parameters/Blob"))});
+        } 
+        else if (property.includes('/{containerName}'))
+        {
+            $[property]["parameters"] = $[property]["parameters"].filter(function(param) { return (typeof param['$ref'] === "undefined") || (false == param['$ref'].endsWith("#/parameters/ContainerName"))});
+        }
```

Comment:
what does it do ? Could you add paragraph header explaining what this transform does?

Reply:
`Don't include container name or blob in path - we have direct URIs.`

Fixed.

Reply:
@kasobol-msft - this transform is ensuring to remove the container and blob parameters from the parameter list. This prevents it from being required on an operation level. This transform was needed for Python and .NET since we don't construct the path on the operation level. So we don't need those params.